analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -209,12 +209,12 @@ def filter_compiler_includes_extra_args(compiler_flags):
 
     compiler_flags -- A list of compiler flags which may affect the list
                       of implicit compiler include paths, like -std=,
-                      --sysroot=, -m32, -m64 or -stdlib=.
+                      --sysroot=, -m32, -m64, -nostdinc or -stdlib=.
     """
     # If these options are present in the original build command, they must
     # be forwarded to get_compiler_includes and get_compiler_defines so the
     # resulting includes point to the target that was used in the build.
-    pattern = re.compile('-m(32|64)|-std=|-stdlib=')
+    pattern = re.compile('-m(32|64)|-std=|-stdlib=|-nostdinc')
```

Comment:
will it match for `-nostdinc++` too, we should keep that too right? See: https://gcc.gnu.org/onlinedocs/cpp/Invocation.html

Reply:
Yes, it will, because we use `re.match()` which does a prefix match.